tests/operators_test.py

```diff
@@ -27,7 +28,8 @@ def __init__(self, name):
     def setup_streams(input_streams):
         return [
             DataStream(name='data_stream'),
-            DataStream(name='tuple_data_stream')
+            DataStream(name='tuple_data_stream'),
+            DataStream(name='list_data_stream')
```

Comment:
Just add a trailing comma at the end of the list, and you won't run into changes across two lines to add a single element later.